### DISCUSSION TITLE
Android touch screen controls, multimeter

### DISCUSSION THREAD

**Discussion by Mariovilb** [espotek-org/Labrador#278] [General]
Hi! 

I can't find way to change modes in Android smartphone. When started it's in Oscilloscope mode and no way to change to Multimeter or Signal generator. 
Some special touch, pinch etc for control from touchscreen?

Oneplus 3,  Android 9.0.6

**Comment by EspoTek** [comment]
Hi mate, you just need to swipe from left to right to change pages.  The input's a little bit fiddly, though (Qt on Mobile was a bad idea..) - you'll need to get the position and velocity of the swipe just right.  A short, fast swipe over one of the labels usually prevents the input capture going bad.
